.kitchen.yml

```diff
@@ -82,3 +82,22 @@ suites:
     encrypted_data_bag_secret_key_path: "test/integration/forwarder/encrypted_data_bag_secret"
     run_list:
       - recipe[elkstack::forwarder]
+
+<% if ENV['RACKSPACE_MONITORING_USERNAME'] && ENV['RACKSPACE_MONITORING_API_KEY'] %>
+<%# Disabling backup as elasticsearch_backup seems to fail if you don't give the good credentials %>
+  - name: monitoring # server
+    run_list:
+      - recipe[java::default]
+      - recipe[elkstack::default]
+    attributes:
+      elkstack:
+        config:
+          backups:
+            enabled: false
```

Comment:
I've disabled backup when testing monitoring because : 
* I was getting an error once rackspace::cloud_credentials was set up (for monitoring), it might be because I was using LON credentials which are not supported by the backup recipe.
* it's faster